import-export-cli/integration/apim/apim.go

```diff
@@ -608,6 +608,63 @@ func (instance *Client) AddAPIProductFromJSON(t *testing.T, path string, usernam
 	return apiProductResponse.ID
 }
 
+// CreateAPIRevision : Create API revision
+func (instance *Client) CreateAPIRevision(apiID string) *APIRevision {
+	apiProductsURL := instance.publisherRestURL + "/apis/" + apiID + "/revisions"
+
+	request := base.CreatePost(apiProductsURL, bytes.NewBuffer([]byte("{ \"description\": \"\" }")))
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.CreateAPIRevision()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.CreateAPIRevision()", response, 201)
+
+	var revision APIRevision
+	json.NewDecoder(response.Body).Decode(&revision)
+
+	return &revision
+}
+
+// DeployAPIRevision : Deploy API revision
+func (instance *Client) DeployAPIRevision(t *testing.T, apiID string, revision *APIRevision) {
+	apisURL := instance.publisherRestURL + "/apis/" + apiID + "/deploy-revision"
```

Comment:
Lets change `apisURL` to `deployURL`

Reply:
I think I have made the same mistake in DeployAPIProductRevision(), please fix it for me :)